registry/quilt_server/docker_config.py

```diff
@@ -9,39 +9,10 @@
 
 AUTH_PROVIDER = os.getenv('AUTH_PROVIDER', 'quilt')
 
-QUILT_AUTH_URL = 'http://auth:5002'
-
-if AUTH_PROVIDER == 'quilt':
-    OAUTH = dict(
-        access_token_url='%s/o/token/' % QUILT_AUTH_URL,
-        authorize_url='%s/o/authorize/' % QUILT_AUTH_URL,
-        client_id=os.getenv('OAUTH_CLIENT_ID'),
-        client_secret=os.getenv('OAUTH_CLIENT_SECRET'),
-        user_api='%s/accounts/api-root' % QUILT_AUTH_URL,
-        profile_api='%s/accounts/profile?user=%%s' % QUILT_AUTH_URL,
-        have_refresh_token=True,
-    )
-elif AUTH_PROVIDER == 'github':
-    OAUTH = dict(
-        access_token_url='https://github.com/login/oauth/access_token',
-        authorize_url='https://github.com/login/oauth/authorize',
-        client_id=os.getenv('OAUTH_CLIENT_ID_GITHUB', '411a75cc2b4f6669a418'),
-        client_secret=os.getenv('OAUTH_CLIENT_SECRET_GITHUB', os.getenv('OAUTH_CLIENT_SECRET')),
-        user_api='https://api.github.com/user',
-        profile_api='https://api.github.com/users/%s',  # NO trailing slash
-        have_refresh_token=False,
-    )
-else:
-    assert False, "Unknown auth provider: %s" % AUTH_PROVIDER
-
-OAUTH.update(dict(
-    redirect_url='http://flask:5000/oauth_callback',
-))
-
 CATALOG_URL = 'http://localhost:3000'
 
-# TODO: move invite sending to flask
-INVITE_SEND_URL = '%s/pkginvite/send/' % QUILT_AUTH_URL
+REGISTRY_HOST = 'localhost:5000'
```

Comment:
@meffij why are you hardcoding the urls in docker config? I believe we should use env as much as possible here, because it's very hard to configure things otherwise